projects/demo-playwright/tests/kit/input-date/input-date.spec.ts

```diff
@@ -0,0 +1,140 @@
+            await tuiGoto(page, '/components/input-date');
+
+            const api = new TuiDocumentationPagePO(page);
+            const example = api.getExample('#sizes');
+
+            await example.scrollIntoViewIfNeeded();
+            await api.prepareApiPageBeforeScreenshot(':not(#sizes)');
+
+            for (const size of ['s', 'm', 'l']) {
+                const input = example
+                    .locator(`[tuiTextFieldSize="${size}"]`)
+                    .locator('[automation-id="tui-primitive-textfield__native-input"]');
+
+                await input.click();
+                await expect(page).toHaveScreenshot(`01-01-input-date-${size}.png`);
+                await input.clear();
+                await expect(page).toHaveScreenshot(`01-02-input-date-${size}.png`);
+                await input.focus();
+                await page.keyboard.type('01.');
+                await expect(page).toHaveScreenshot(`01-03-input-date-${size}.png`);
+                await input.focus();
+                await page.keyboard.type('06.1994');
+                await expect(page).toHaveScreenshot(`01-04-input-date-${size}.png`);
+            }
+        });
+    });
+
+    test.describe('API', () => {
+        let api: TuiDocumentationPagePO;
+        let input: Locator;
+
+        test.beforeEach(({page}) => {
+            api = new TuiDocumentationPagePO(page);
+            input = api.apiPageExample.locator(
+                '[automation-id="tui-primitive-textfield__native-input"]',
+            );
+        });
+
+        ['s', 'm', 'l'].forEach(size => {
+            test(`correct filler display for size=${size}`, async ({page}) => {
+                await tuiGoto(
+                    page,
+                    `/components/input-date/API?tuiTextfieldSize=${size}`,
+                );
+
+                await input.click();
+                await api.prepareApiPageBeforeScreenshot();
+                await expect(page).toHaveScreenshot(`02-01-input-date-${size}.png`);
+                await page.keyboard.type('01.');
+                await expect(page).toHaveScreenshot(`02-02-input-date-${size}.png`);
+                await page.keyboard.type('06.1994');
+                await expect(page).toHaveScreenshot(`02-03-input-date-${size}.png`);
+            });
+        });
+
+        test('maximum month less than current month', async ({page}) => {
+            await tuiGoto(page, '/components/input-date/API?max$=1');
+
+            await input.click();
+            await api.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot('03-input-date.png');
+        });
+
+        test('minimum month more than current month', async ({page}) => {
+            await tuiGoto(page, '/components/input-date/API?min$=3');
+
+            await input.click();
+            await api.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot('04-input-date.png');
```

Comment:
Let's decrease the size of screenshot

![04-input-date-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/049dc15c-c36a-4deb-90e8-18a1126ae299)

Reply:
fixed